Webshop.Catalog.Solution/ProcessingService/Program.cs

```diff
@@ -0,0 +1,31 @@
+﻿using RabbitMQ.Client;
+using RabbitMQ.Client.Events;
+using System;
+using System.Text;
+
+class Program
+{
+    static void Main()
+    {
+        var factory = new ConnectionFactory() { HostName = "localhost" };
+
+        using (var connection = factory.CreateConnection())
+        using (var channel = connection.CreateModel())
+        {
+            channel.QueueDeclare(queue: "your_queue_name", durable: false, exclusive: false, autoDelete: false, arguments: null);
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🤓 nitpick (non-blocking):** ‏If you take the above comment into consideration, don't forget to update it here as well. :P

Reply:
yes sir